www/index.html

```diff
@@ -103,66 +121,70 @@
         function geolocationError(error) {
             console.warn('ERROR(' + error.code + '): ' + error.message);
 
-            var toast = document.querySelector('paper-toast');
-            toast.text = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
+            var toast      = document.querySelector('paper-toast');
+            toast.text     = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
             toast.duration = 5000;
```

Comment:
I think that one element = one role, so having many toast elements seems legit.
Moreover, it's more maintenable to find the content into the markup instead of into the js part (which should serve only to trigger the notification).